node/src/components/consensus/era_supervisor.rs

```diff
@@ -660,15 +694,15 @@ where
 
             let slashed = self
                 .era_supervisor
-                .iter_past_other(era_id, self.era_supervisor.bonded_eras)
+                .iter_past(era_id, self.era_supervisor.bonded_eras)
```

Comment:
Interesting change, can you explain?

Reply:
It's because of the change in semantics of `switch_blocks` - the last interesting switch block is the one for the current era, which was under the key `era_id - 1` before (because it is the switch block of the era `era_id-1`), and is under `era_id` now.

Reply:
Ah, right. I realized that some time alter. Thanks.